cdap-ui/app/cdap/components/Cloud/Profiles/Store/ActionCreator.js

```diff
@@ -127,7 +127,7 @@ export const fetchAggregateProfileMetrics = (namespace, profile, extraTags) => {
               if (metric.data.length === 1) {
                 metricValue = metric.data[0].value;
               } else {
-                metricValue = metric.data.reduce((prev, curr) => prev + curr.value);
+                metricValue = metric.data.reduce((prev, curr) => prev + curr.value, metric.data[0].value);
```

Comment:
Just want to confirm, it's `prev + curr.value`, not `prev.value + curr.value`?

Reply:
No its just adding numbers.